test/utils/browser.js

```diff
@@ -2,22 +2,24 @@
 var assert = require('assert');
 var fs = require('fs');
 var functionToString = require('function-to-string');
-var async = require('async');
 var wd = require('wd');
 
-// Resolve cookies with helpful messaging
-var cookiesJson;
-try {
-  cookiesJson = fs.readFileSync(__dirname + '/../cookies.json');
-} catch (err) {
-  throw new Error('Could not read `test/cookies.json`. Please make sure it exists. ' +
-      'If it doesn\'t, follow the steps in https://github.com/twolfson/google-music.js#testing');
-}
-var cookies = JSON.parse(cookiesJson);
-
 // Resolve the compiled script
 var script = fs.readFileSync(__dirname + '/../../dist/google-music.js', 'utf8');
 
+// Google Login Info you have to fill this in
+var username = process.env.GPM_USER;
+var password = process.env.GPM_PWD;
+
+// These are the BrowserStack username and key vars that should NEVER be
+// pushed to remote.  They are stored in secure env vars on travis.ci
+var BS_USER = process.env.BS_USER;
+var BS_KEY = process.env.BS_KEY;
+
+if (!username || !password) {
+  throw 'You need to define your GPM Username and Password';
+}
```

Comment:
Instead of using these lines, please leverage `assert` like the rest of this script:

```js
// Google Login Info you have to fill this in
var username = process.env.GPM_USER;
var password = process.env.GPM_PWD;
assert(username, 'Username for Google Music wasn\'t set. Please set it via the GPM_USER environment variable');
assert(password, 'Password for Google Music wasn\'t set. Please set it via the GPM_PWD environment variable');
```